LICENSE

```diff
@@ -0,0 +1,21 @@
+The Weaver is licensed under the open source [MIT
+License](http://opensource.org/licenses/MIT).
+
+Copyright (c) 2011 Weecology
```

Comment:
Update the copyright date to 2015 and replace Weecology with "the Data Weaver contributors and the University of Florida".